qubes/vm/mix/dvmtemplate.py

```diff
@@ -34,6 +38,88 @@ class DVMTemplateMixin(qubes.events.Emitter):
         doc="Should this VM be allowed to start as Disposable VM",
     )
 
+    def get_feat_preload(self) -> list:
+        feature = "preload-dispvm"
+        # TODO: fix, mypy throws:
+        # error: "DVMTemplateMixin" has no attribute "features"  [attr-defined]
```

Comment:
mypy doesn't understand mixins... there are two options:
- add `assert isinstance(self, BaseVM)` (or similar)
- ignore (as you did now)